packages/react/src/QueeryKeeper.spec.tsx

```diff
@@ -0,0 +1,26 @@
+import { FronteggProvider } from './FronteggProvider';
+import React from 'react';
+import { render } from '@testing-library/react';
+import { BrowserRouter } from 'react-router-dom';
+import * as QueryKeeper from './queryKeeper';
+import 'whatwg-fetch';
+
+describe('Query keeper', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('Should call useQueryKeeper when no external history is provided ', async () => {
+    const mockUseQueryKeeper = jest.spyOn(QueryKeeper, 'useQueryKeeper');
```

Comment:
I think a better approach is to check some logic (like testing history.push is called). I think that `useQueryKeeper` is just an internal API. I would want this test (and other) to work also when the API is changing. The important part is the logic itself. But, it's an approach. This is my thoughts. You can also apply them both.